v8-sys/src/v8-glue.cc

```diff
@@ -899,11 +900,11 @@ ValueRef v8_Object_CallAsFunction(RustContext c, ObjectRef self, ContextRef cont
     v8::HandleScope scope(c.isolate);
     v8::TryCatch try_catch(c.isolate);
     v8::Context::Scope context_scope(wrap(c.isolate, context));
-    v8::Local<v8::Value> argv_wrapped[argc];
+    std::vector<v8::Local<v8::Value>> argv_wrapped(argc);
```

Comment:
It would be so nice to not heap allocate here but I guess we have no choice with portable C++ so :+1: